run-directive/action.yml

```diff
@@ -0,0 +1,54 @@
+name: Run Directive
+run-name: test
+
+inputs:
+  hash:
+    description: 'Github commit hash to checkout'
+    required: false
+    default: ''
+  working-directory:
+    description: 'Name of component directory'
+    required: true
+  config:
+    description: 'Config JSON file'
+    required: true
+  stage:
+    description: 'Name of stage'
+    required: true
+  directive:
+    description: 'A directive to run. e.g. npm ci, npm run test'
+    required: true
+
+jobs:
+  run-directive:
+    name: Run Directive
+    using: "composite"
+
+    steps:
+      - name: Setup Environment
+        uses: Brightspace/bdp-ci-action/setup-env@master
```

Comment:
https://github.com/Brightspace/bdp-ci-action/blob/1ccef5a282bde5fcc2be83b027ac74c9e663c648/setup-env/action.yml#L1-L21

🤔 our self-hosted runners should have these and/or should be handling this for us. I think this step can be removed?

Reply:
Interesting.
Since I was working in the bdp-auth-infrastructure repo yesterday, I was looking at the [ci workflow there](https://github.com/Brightspace/bdp-auth-infrastructure/blob/master/.github/workflows/ci.yml) which includes this step. 
Looking at the BDP, this step is not present, so I'll believe you that we don't need it.